com.unity.render-pipelines.high-definition/Runtime/Sky/PhysicallyBasedSky/PhysicallyBasedSky.shader

```diff
@@ -143,7 +143,7 @@ Shader "Hidden/HDRP/Sky/PbrSky"
                         else // Flare region.
                         {
                             float r = max(0, rad - radInner);
-                            float w = saturate(1 - r * rcp(light.flareSize));
+                            float w = max(FLT_MIN, 1 - r * rcp(light.flareSize));
 
                             color *= light.flareTint;
                             scale *= pow(w, light.flareFalloff);
```

Comment:
Shoudln't we use SafePositivePow instead?

i.e 

                float w = saturate(1 - r * rcp(light.flareSize));	 
                            color *= light.flareTint;
                            scale *= SafePositivePow (w, light.flareFalloff);

Reply:
it kind of does the same but maybe it's cleaner so ok